plugin/hover.py

```diff
@@ -210,14 +229,14 @@ def _show_hover(self, listener: AbstractViewListener, point: int, only_diagnosti
 
         if contents:
             if self.view.is_popup_visible():
-                update_lsp_popup(self.view, contents)
```

Comment:
There could be already a pop due a hover which is in a different location from the selection. So we should close it to show the new popup at the right place